python/raml/bld.bat

```diff
@@ -0,0 +1,9 @@
+set INCLUDE=%LIBRARY_INC%
+set LIBPATH=%LIBRARY_LIB%
+set LIB=%LIBRARY_LIB%
+
+python setup.py install
+if errorlevel 1 exit 1
+
+copy %LIBRARY_BIN%\yaml.dll %SP_DIR%\raml\ext\
```

Comment:
@ilanschnell note the `%SP_DIR%\raml\ext\` directory.
compare to: https://bitbucket.org/kalefranz/yaml/src/default/setup.py?fileviewer=file-view-default#setup.py-20